core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1297,6 +1297,12 @@
 
   public static final String CARBON_SESSIONSTATE_CLASSNAME = "spark.carbon.sessionstate.classname";
 
+  /**
+   * This property will be used to configure the sqlastbuilder class.
```

Comment:
Why sqlastbuilder class configuration is required?

Reply:
This required for decouple feature.